SparkPost/2web2ui
# SA-1851: Compare by for links tabs

### What Changed

- Added comparison tabs for links comparison

### How To Test

- Go to report builder and enable compare by feature flag `allow_compare_by`
- Add comparison filters
- Add metrics to generate links tabs (such as `Clicks`)
- Verify that tabs are generated for each comparison
- Click on each tab
- Verify the request results are different for each comparison

### To Do
